src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -127,6 +129,9 @@ Decorator.propTypes = {
     return null;
   },
 
+  /** @type {string} The title of data. */
+  title: PropTypes.string,
```

Comment:
Can you also add this prop to the `DataDecorator` & pass it through to the inner `Decorator`?

Reply:
Added